source/guide_xdebug.rst

```diff
@@ -0,0 +1,149 @@
+.. highlight:: console
+
+.. author:: Sascha Groetzner <https://groetzner.net>
+
+.. categorize your guide! refer to the current list of tags: https://lab.uberspace.de/tags
+.. tag:: lang-php
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/xdebug.svg
+      :align: center
+
+######
+Xdebug
+######
+
+
+.. tag_list::
+
+`Xdebug <https://xdebug.org>`_ is an extension for PHP, and provides a range of features to improve the PHP development experience.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`PHP <lang-php>`
+
+License
+=======
+
+Xdebug is released under `The Xdebug License
+<https://github.com/xdebug/xdebug/blob/master/LICENSE>`_, which is based on
+`The PHP License <https://github.com/php/php-src/blob/master/LICENSE>`_. It is
+an Open Source license (though not explicitly endorsed by the Open Source
+Initiative).
+
+Prerequisites
+=============
+
+We're using :manual:`PHP <lang-php>` in the stable version 8:
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ uberspace tools version show php
+ Using 'PHP' version: '8.0'
+ [isabell@stardust ~]$
+
+Installation
+============
+
+export PHP Path
+---------------
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ export PATH="/opt/remi/php80/root/bin:$PATH"
```

Comment:
This path is subject to change and we'd like to not document it. However, we can expose `phpize` binary just like we already do with the `php` one. Are there any other binaries that are needed to build the extension?

Once that change is released, I'll take your guide for a spin :) Will take a few weeks, though, because we are focusing on a different project right now.

Reply:
> Are there any other binaries that are needed to build the extension?
I don`t think so. `phpize` should be the only one

Reply:
phpize will land in the next U7 release next week-ish :)

Reply:
@groetzner-net `phpize` has been available without special paths for some time now, would you like to update the guide? :)

Reply:
Hi, I have tried it with phpize, however I fail now at the problem that php-config from PHP development headers is also required.

Reply:
we'll expose `php-config` as well, I'll keep you posted.

Reply:
Will be part of the next release (probably later this week).

Reply:
started [rolling out](https://github.com/Uberspace/manual/pull/344/) today